core/src/main/java/tc/oc/pgm/modules/MapmakerMatchModule.java

```diff
@@ -0,0 +1,34 @@
+package tc.oc.pgm.modules;
+
+import java.util.Collection;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.api.map.Contributor;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.match.MatchModule;
+import tc.oc.pgm.api.match.MatchScope;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.api.player.event.MatchPlayerAddEvent;
+import tc.oc.pgm.events.ListenerScope;
+
+@ListenerScope(MatchScope.LOADED)
+public class MapmakerMatchModule implements MatchModule, Listener {
+  private final Collection<Contributor> authors;
+
+  public MapmakerMatchModule(Match match) {
+    this.authors = match.getMap().getAuthors();
+  }
+
+  @EventHandler(ignoreCancelled = true)
+  public void onPlayerAddEvent(final MatchPlayerAddEvent event) {
+    MatchPlayer player = event.getPlayer();
+    player
+        .getBukkit()
+        .addAttachment(
+            PGM.get(),
+            "pgm.group.mapmaker",
```

Comment:
Should define this in `Permissions`

Reply:
Define the permission group? So move it from `config` -> `Permissions`?

Reply:
Then i would also have to code in a prefix provider for mapmakers?

Reply:
All that's being asked is for the constant string `pgm.group.mapmaker` to be defined in the `Permissions.java` file instead of here

Reply:
Aaaah, i see. Thanks!